src/charm.py

```diff
@@ -1,187 +1,201 @@
 #!/usr/bin/env python3
 
-import sys
-sys.path.append('lib')
+import logging
+import random
+import string
+import math
+import re
 
 from base64 import b64encode
-from ops.charm import CharmBase, CharmEvents
-from ops.framework import EventSource, EventBase, StoredState
+
+from ops.charm import CharmBase
 from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus
 from ops.main import main
+from charmhelpers.core.hookenv import (
+    leader_get,
+    leader_set,
```

Comment:
in our call we recommended dropping `leader_get`/`leader_set` in favour a peer relation. I was hoping to see that change here.

Reply:
Fixed in the latest update.